bobtimus/src/lib.rs

```diff
@@ -303,6 +300,11 @@ where
         );
         let oracle_pk = PublicKey::from_private_key(&self.secp, &oralce_priv_key);
 
+        // calculate the absolute timelock from the loan term
+        let current_height = self.elementsd.get_blockcount().await?;
+        let term_in_minutes = payload.term * 24 * 60;
```

Comment:
I think the loan term should be a unix timestamp. Nothing to do in this PR but I recorded it here: https://github.com/comit-network/baru/issues/41

Reply:
I gave this a shot in https://github.com/comit-network/waves/pull/208/commits/a958f8409f11271f58c2d144ea0596be0bcaef2d